{{cookiecutter.github_repo}}/scripts/deploy/fetch-scripts.sh

```diff
@@ -0,0 +1,3 @@
+#! /bin/bash
+
+git clone https://github.com/bhgrant8/deploy-scripts.git && mv ./deploy-scripts/scripts/deploy/* ./scripts/deploy
```

Comment:
```suggestion
git clone https://github.com/hackoregon/deploy-scripts.git && mv ./deploy-scripts/scripts/deploy/* ./scripts/deploy
```

Is this a fork of the main repo? Do we need to PR/merge some changes to the main repo?

Reply:
yeah, @nam20485 if you want to review: https://github.com/hackoregon/deploy-scripts/pull/3
then we can commit your suggestion (I was just using the other fork as temporary example)

Reply:
I have reviewed and approved the deploy-scripts changes. I do feel that @DingoEatingFuzz's suggestion is a good one, so once we merge this in and test that it works I can add the one more level to account for frontend-specific scripts etc.